airbyte-integrations/connectors/source-hubspot/source_hubspot/api.py

```diff
@@ -501,6 +540,79 @@ def read_chunked(
             yield from super().read(getter, params)
 
 
+class CRMSearchStream(IncrementalStream, ABC):
+
+    limit = 100  # This value is used only when state is None.
+    state_pk = "updatedAt"
+    updated_at_field = "updatedAt"
+    
+    @property
+    def url(self):
+        if self.state:
+            return f"/crm/v3/objects/{self.entity}/search"
+        return f"/crm/v3/objects/{self.entity}"
+
+    def __init__(
+        self,
+        entity: Optional[str] = None,
+        last_modified_field: Optional[str] = None,
+        associations: Optional[List[str]] = None,
+        include_archived_only: bool = False, **kwargs
+    ):
+        super().__init__(**kwargs)
+        self._state = None
+        self.entity = entity or self.entity
+        self.last_modified_field = last_modified_field or self.last_modified_field
+        self.associations = associations or self.associations
```

Comment:
```suggestion
        self.entity = entity
        self.last_modified_field = last_modified_field
        self.associations = associations
```
I think you did not set these class variables.  `CRMObjectStream` does.